File: Goal.java
public class Goal {
    private float goal;
    private float savings;

    public Goal(float goal, float savings) {
        this.goal = goal;
    }

    public void setGoal(float goal) {
        this.goal = goal;
    }

    public float getGoal() {
        return goal;
    }

    public void getSavings(float savings) {
        this.savings = savings;
    }

    public float getSavings() {
        return savings;
    }

    public boolean validateGoal(float goal) {
        return false;
    }

}

File: src/Goal.js
class Goal {
    constructor(goal, savings) {
        this.goal = goal;
        this.savings = savings;
    }

    setGoal(goal) {
        this.goal = goal;
    }

    getGoal() {
        return this.goal;
    }

    setSavings(savings) {
        this.savings = savings;
    }

    getSavings() {
        return this.savings;
    }

    validateGoal(goal) {
        return false; 
    }
}

export default Goal;
